usr/src/cmd/svc/milestone/net-physical

```diff
@@ -395,6 +395,26 @@ function setup_mtu
     done
 }
 
+#
```

Comment:
Suggest you initialize RESOLV_CONF_DONE here to "alse" or something else just to keep the casual reader from being surprised.